src/view/mod.rs

```diff
@@ -46,8 +46,14 @@ pub struct View {
 }
 
 impl View {
-    pub fn new(preferences: Rc<RefCell<Preferences>>, event_channel: Sender<Event>) -> Result<View> {
-        let terminal = build_terminal().chain_err(|| "Failed to initialize terminal")?;
+    /// Return a new view. This will initialize the terminal, load the theme,
+    /// and begin to listen for events.
+    pub fn new(
+        preferences: Rc<RefCell<Preferences>>,
+        event_channel: Sender<Event>
+    ) -> Result<View> {
+        let terminal = build_terminal()
+                       .chain_err(|| "Failed to initialize terminal")?;
```

Comment:
Yeah sorry, I was playing with the codebase a bit trying to understand it, and I think I accidentally commited this instead of starting from the original codebase when I tried to implement line wrapping.